### PR TITLE
Twilio channel auth

### PR DESCRIPTION
**Pull request by nicpottier**
Sets our auth information on each channel so that courier can access them

### CODE REVIEW

**Comment by nicpottier** [comment]
@norkans7 so you are talking about someone removing their Twilio configuration? I think that's actually ok and by design, in that we want to enable people having channels active from multiple twilio accounts.

**Comment by norkans7** [comment]
Yep if they remove or change the config the channel will not be updated, 
If we want to support many accounts on one org, then we move the form for the config in the channels app and that configuration is not saved to the org

**Comment by nicpottier** [comment]
I think the current behavior currently is correct, yes, you can remove it and we'll continue using credentials for when you first connected it. I'm ok with that.

File: temba/channels/migrations/0079_auto_20170910_1731.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    for channel in Channel.objects.filter(channel_type__in=['T', 'TMS'], is_active=True).select_related('org'):
+        config = json.loads(channel.config)
+        org_config = json.loads(channel.org.config)
+        config['account_sid'] = org_config.get('ACCOUNT_SID')
+        config['auth_token'] = org_config.get('ACCOUNT_TOKEN')
+        channel.config = json.dumps(config)
+        channel.save(update_fields=['config'])
+
+    # for consistency, remap twilio message service keys as well
+    for channel in Channel.objects.filter(channel_type=['TW'], is_active=True):
```
Comment (rowanseymour) [inline temba/channels/migrations/0079_auto_20170910_1731.py]: channel_type='TMS'

Comment (rowanseymour) [reply temba/channels/migrations/0079_auto_20170910_1731.py]: Wait no.. `channel_type='TW'` and fix comment

**Pull request merged**